tests/kernel/bigquery_tests.py

```diff
@@ -773,82 +766,24 @@ def test_pipeline_cell(self, mock_get_table, mock_table_exists, mock_environment
       }
       mock_environment.return_value = env
 
-      args = {'name': 'bq_pipeline_test', 'debug': True, 'billing': 'foo_billing'}
-      # TODO(rajivpb): The references to foo_query need to be resolved.
-      cell_body = """
+      args = {'name': 'bq_pipeline_test'}
+      small_cell_body = """
+              email: foo1@test.com
               schedule:
                   start: 2009-05-05T22:28:15Z
                   end: 2009-05-06T22:28:15Z
                   interval: '@hourly'
               input:
-                  path: test/path
                   table: project.test.table
-                  schema:
-                      - name: col1
-                        type: int64
-                        mode: NULLABLE
-                        description: description1
-                      - name: col2
-                        type: STRING
-                        mode: required
-                        description: description1
               transformation:
                   query: foo_query
               output:
-                  path: test/path
                   table: project.test.table
          """
 
-      output = google.datalab.contrib.bigquery.commands._bigquery._pipeline_cell(args, cell_body)
```

Comment:
It's not clear to me what these changes to the test are. Is it not testing the pipeline anymore?

Reply:
I'm trying to limit the use of the long "golden" file string to as few tests as possible (ideally just two - one for %bq and one for %pipeline. This was a test that used it unnecessarily and had substantial overlap with those tests, so I did some refactoring and shortened this.